domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -589,6 +603,10 @@ class ExplorationProgressController @Inject constructor(
     saveExplorationCheckpoint()
   }
 
+  /** Checks if the answer was submitted to the state on behalf of continue interaction. */
```

Comment:
```suggestion
  /** Returns whether the specified interaction automatically continues the user to the next state upon completion. */
```

Needs line wrapping.

Reply:
done

Reply:
Marking as resolved as already done